frontend/.env

```
@@ -0,0 +1,2 @@
+REACT_APP_SERVER_BASE_ENDPOINT_DEV=http://localhost:8000/api
+REACT_APP_SERVER_BASE_ENDPOINT_PROD=http://localhost:8000/api
```

```suggestion
REACT_APP_SERVER_BASE_ENDPOINT_PROD=http://dataviz.creativecommons.engineering/api
```

This needs to be the real value to automate the frontend build.